src/js/components/UserInformation/Address.jsx

```diff
@@ -14,6 +14,7 @@ class Address extends Component {
   }
 
   mouseOverAddress = () => {
+    if(this.props.disableHover) return;
```

Comment:
I'd move that logic to the `onMouseOver` of the component, that way we don't add a listener for no reasons